content/concepts/authentication/content.md

```diff
@@ -175,3 +175,29 @@ podIdentity:
 ```
 
 Azure AD Pod Identity will give access to containers with a defined label for `aadpodidbinding`.  You can set this label on the KEDA operator deployment.  This can be done for you during deployment with Helm with `--set aadPodIdentity={your-label-name}`.
+
+#### Kiam Pod Identity for AWS
+
+[**Kiam**](https://github.com/uswitch/kiam/) lets you bind an AWS IAM Role to a pod using an annotation on the pod.
+
+You can tell KEDA to use Kiam via `podIdentity.provider`.
+
+```yaml
+podIdentity:
+  provider: kiam # Optional. Default: false
+```
+
+Kiam will give access to containers based on the role provided by the `iam.amazonaws.com/role` annotation on the pod.
+
+#### EKS Pod Identity Webhook for AWS
```

Comment:
Personally I'd prefer to move the official project before the community approach, but it's up to you.